qiskit/circuit/measure.py

```diff
@@ -22,7 +22,10 @@ class Measure(SingletonInstruction):
     """Quantum measurement in the computational basis."""
 
     def __init__(self, label=None, *, duration=None, unit="dt"):
-        """Create new measurement instruction."""
+        """
+        Args:
+            label: optional string label for this instruction.
```

Comment:
should `duration` and `unit` be added too? Or are them inherited?

Reply:
They're not inherited, but we also haven't really documented them in any of the other instruction constructors.  If possible, I'd potentially like to leave changing the documentation around the calibration/scheduling-related components for another PR, if that's ok?  I think there's _a lot_ more work to be done about that, which I largely haven't touched.